tests/unit/returners/test_sentry.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+
+# Import Python libs
+from __future__ import absolute_import
+
+# Import Salt Testing libs
+from tests.support.unit import TestCase
+
+# Import salt libs
+import salt.returners.sentry_return as sentry
+
+
+class SentryReturnerTestCase(TestCase):
+    '''
+    Test Sentry Returner
+    '''
+    ret = {'id': '12345',
+           'fun': 'mytest.func',
+           'fun_args': ['arg1', 'arg2', {'foo': 'bar'}],
+           'jid': '54321',
+           'return': 'Long Return containing a Traceback'}
+
+    def test_get_message(self):
+        assert sentry._get_message(self.ret) == 'salt func: mytest.func arg1 arg2 foo=bar'
```

Comment:
Hey @nasenbaer13 - Unfortunately we haven't made the full switch over to pytest. Can you change these assertions from `assert` to the [unittest assertion types](https://docs.python.org/2/library/unittest.html#assert-methods) such as `assertEqual`?

Reply:
Done